common/lib/xmodule/xmodule/tests/test_word_cloud.py

```diff
@@ -38,15 +38,15 @@ def test_good_ajax_request(self):
             response['student_words'],
             {'sun': 1, 'dog': 6, 'cat': 12}
         )
-        self.assertListEqual(
-            response['top_words'],
-            [{'text': 'dad', 'size': 2, 'percent': 9.0},
-             {'text': 'sun', 'size': 1, 'percent': 5.0},
-             {'text': 'dog', 'size': 6, 'percent': 27.0},
-             {'text': 'mom', 'size': 1, 'percent': 5.0},
-             {'text': 'cat', 'size': 12, 'percent': 54.0}]
-        )
 
-        self.assertEqual(
-            100.0,
-            sum(i['percent'] for i in response['top_words']))
+        expected_top_words = [{'text': 'dad', 'size': 2, 'percent': 9},
```

Comment:
It looks like this used to be floating point numbers but now they are integers, what's the reason for that?

Reply:
values are back to be floating-point numbers.